docs/apache-airflow/concepts/dags.rst

```diff
@@ -796,39 +796,32 @@ the dependency graph.
 The dependency detector is configurable, so you can implement your own logic different than the defaults in
 :class:`~airflow.serialization.serialized_objects.DependencyDetector`
 
-DAG pausing, deactivation and deletion
---------------------------------------
+Unschedule or Delete a DAG
```

Comment:
Unschedule is a strange word.

Reply:
How about "deschedule" ? That sounds much better.

Reply:
No strong feelings either way for me! Deschedule probably matches the latin roots better? Updated accordingly

Reply:
Plus the alliteration is nice 😄

Reply:
Disable?